lib/ava-error.js

```diff
@@ -9,4 +9,6 @@ function AvaError(message) {
 	this.name = 'AvaError';
 }
 
+AvaError.prototype = Object.create(Error.prototype);
```

Comment:
Needed to make `.catch(err, handler)` work. Without this change, Bluebird did not detect `AvaError` class as a class inherited from `Error` and assumed it was `handler`.